### DISCUSSION TITLE
Need information about the library or the gotenberg php package

### DISCUSSION THREAD

**Discussion by al3xanderSL** [gotenberg/gotenberg#775] [Q&A]
Hello, I have just discovered you library which looks amazing, and I would like to know if gotenberg is compatible with Laravel 10, and php 8.3, besides, I would  also need to know if gotenberg supports tailwindcss styles to generate pdf.

**Comment by gulien** [comment]
It supports PHP 8.3, and I guess it works with Laravel.
No idea about tailwind.

**Comment by IgordeOliveira** [comment]
I'm using Laravel and tailwind, but I needed to add the tailwind as CDN inside the html, I do not know if it is the best option
